examples/s2_field_combination_extern_rf_train/re_add_raster_dataset_from_upload.ipynb

```diff
@@ -0,0 +1,5550 @@
+{
```

Comment:
Please remove this file or format it with headings plus descriptions.
Moreover, there are local paths from your PC.

Reply:
![grafik](https://user-images.githubusercontent.com/4947061/232978525-77627381-b621-4d6e-a113-86d7b70b42a2.png)

You comittet a lot of binary files that are about 40-45 MB in size.
Please remove them and then we should remember to squash-merge these commits.

Reply:
[e14218e](https://github.com/geo-engine/geoengine-python/pull/119/commits/e14218eb97983d9208e2499b0b9cb8eb10ef6cb3)